app/over_react_redux/todo_client/lib/src/components/app_bar/app_bar_theme_menu.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:over_react/over_react.dart';
+import 'package:over_react/over_react_redux.dart';
+
+import 'package:todo_client/src/actions.dart';
+import 'package:todo_client/src/store.dart';
+import 'package:react_material_ui/react_material_ui.dart';
+import 'package:todo_client/src/components/shared/menu_overlay.dart';
+
+part 'app_bar_theme_menu.over_react.g.dart';
```

Comment:
This could just use an ignore statement!

```suggestion
part 'app_bar_theme_menu.over_react.g.dart';  // ignore: uri_has_not_been_generated
```